src/R/Editor/Impl/Data/WorkspaceVariableProvider.cs

```diff
@@ -85,7 +85,7 @@ internal sealed class WorkspaceVariableProvider : RSessionChangeWatcher, IVariab
                 }
 
                 // May be a package object line mtcars$
-                variableName = TrimToTrailingSelector(variableName);
+                variableName = TrimToFirstSelector(variableName);
```

Comment:
For the future, it would be nice to refactor this code so that it doesn't try to distinguish between globals and members at all. Globals are really just members of the global environment object, and so you can start with `.GlobalEnv` as root expression, and then use `DescribeChildrenAsync` to get them. Then the entire code snippet below will be uniform in that regard, with no `exists()` check needed at all.